extra-network/clash/autobuild/defines

```diff
@@ -3,4 +3,3 @@ PKGSEC=net
 PKGDEP="glibc"
 BUILDDEP="go"
 PKGDES="A rule-based tunnel for VMess, Shadowsocks, Trojan, Snell protocols"
-ABTYPE=gomod
```

Comment:
Did this somehow not work with your first PR that introduced this package?

Reply:
Because `gomod` cannot pass the version message to `clash`. Then, I use the `build` file.